rubin_jupyter_utils/hub/scanrepo/reaper.py

```diff
@@ -177,56 +177,3 @@ def _delete_tags_from_docker_hub(self):
                 del self._results_map[t]
             if self.cachefile:
                 self._writecachefile()  # Remove deleted tags
-
```

Comment:
Took me a while but I understand why this needs to move from reaper to a more central place, because now we have to authenticate to pull as well.